### ISSUE TITLE
Storage Spaces [non-Direct] ?

### ISSUE THREAD

**Issue by evilensky** [MicrosoftDocs/windowsserverdocs#650]
This document includes this statement:

>Storage Spaces integration - When used in conjunction with a mirror or parity space, ReFS can automatically repair detected corruptions using the alternate copy of the data provided by Storage Spaces. Repair processes are both localized to the area of corruption and performed online, requiring no volume downtime.

Where can one find the documentation on Storage Spaces [non-Direct] for Server 2016+.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: eb95c569-f9ad-8e2d-3423-2c9629e65b72
* Version Independent ID: e2a4c64a-97c0-b310-2e47-d0c348a8ead7
* Content: [Resilient File System (ReFS) overview](https://docs.microsoft.com/en-us/windows-server/storage/refs/refs-overview)
* Content Source: [WindowsServerDocs/storage/refs/refs-overview.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/storage/refs/refs-overview.md)
* Service: **unspecified**
* GitHub Login: @gawatu
* Microsoft Alias: **gawatu**

**Comment by lizap** [comment]
@JasonGerend - can you answer this question? Thanks!

**Comment by JasonGerend** [comment]
I wrote a short [Storage Spaces overview](https://docs.microsoft.com/en-us/windows-server/storage/storage-spaces/overview) topic to help readers find the Storage Spaces content they need, and added links to a few topics, including this ReFS topic. The new overview topic links to our Windows Server 2012 R2 content on Shared SAS and stand-alone Storage Spaces, neither of which we've updated yet, but the content should work fine with Windows Server 2016. I hope this helps!
